ConvertParse.py

```diff
@@ -0,0 +1,108 @@
+def convert_parse_labs(name,course):
+	labs = course['lab hours'] if 'lab hours' in course else 0
+	if labs == 'at least 9 hours per semester':
```

Comment:
This is ok for now, but we might think about parsing these strings into `at least X hours per semester` or `at least X Y-hour sessions per semester` using the [re](https://docs.python.org/2/library/re.html) library.